### DISCUSSION TITLE
namelist_examples

### DISCUSSION THREAD

**Discussion by smallrainbow** [SmileiPIC/Smilei#766] [General]
Dear developers,

For the latest v5.1 version, is there some namelist of magnetic reconnection that can be used for learning?
For example, in the examples mentioned in highlights https://smileipic.github.io/Smilei/Overview/highlights.html
Thank you very much!

Best regards,
Zhongwei

**Comment by charlesprouveur** [comment]
Hello, 
This "issue" was converted into a conversation since this is not an issue.
Charles
Note that you can ask such questions directly in https://app.element.io/?updated=1.11.31#/room/#Smilei-users:matrix.org

**Comment by beck-llr** [comment]
Hello Zhongwei

In the future, please use "Discussions" instead of "issues" for this kind of question please.
Some authors of magnetic reconnection papers with Smilei answer questions on the chat. I suggest to ask your question over there.
Since this kind of simulation is usually pretty large, we do not plan to include these in the benchmarks for the moment even though I agree it would be convenient.

**Reply by smallrainbow** [reply]
Ok, I see. Thank you for the reminding.
